CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 1.5.0 (released to-be-determined)
+
+- Removed support for Python 2
```

Comment:
minor: dots at the end of enumerations to be consistent with the rest of the file.